### ISSUE TITLE
Bottom bar position 

### ISSUE THREAD

**Issue by pfr-dev** [kolunmi/sandbar#19]
Is there an option to set the bar at the bottom of the screen?

**Comment by realcharmer** [comment]
I agree that the examples need improving. You can set your bar to be on the bottom by appending `-bottom` when running sandbar, for example:

```
while cat "$FIFO"; do :; done | sandbar \
	-font "LiterationMono Nerd Font:size=16" \
	-active-bg-color "#9D3232" \
	-inactive-bg-color "#222222" \
	-title-bg-color "#222222" \
	-bottom
```

**Comment by kolunmi** [comment]
@pfr-dev 

You can set the bar at the bottom by passing in `-bottom` on invocation like @realcharmer suggested, or you can write `all set-bottom` / `all set-top` into sandbar's stdin to change it at runtime. If you have a fifo setup like the one shown in the readme, you can do this:
```sh
echo 'all set-bottom' > "$MY_FIFO_PATH"
```
you can replace `all` with your specific output name if you want

realcharmer answered your second question, just pass `-font "${MY_FONT_NAME}:size=${MY_FONT_SIZE}"` as an argument

As for your third question, the `-no-title` option exists to disable the window title, but nothing for the wm mode and layout. For now, you can probably just comment out these lines: https://github.com/kolunmi/sandbar/blob/main/sandbar.c#L412-L423 and you'll get the effect you want

the example on the readme was contributed. What in particular could use improvement?

**Comment by pfr-dev** [comment]
> I agree that the examples need improving. You can set your bar to be on the bottom by appending `-bottom` when running the `sandbar` binary. Font size is set by adding `:size=n` to the font name. For example:
> 
> ```
> while cat "$FIFO"; do :; done | sandbar \
> 	-font "LiterationMono Nerd Font:size=16" \
> 	-active-bg-color "#9D3232" \
> 	-inactive-bg-color "#222222" \
> 	-title-bg-color "#222222" \
> 	-bottom
> ```

This was very helpful, thanks! Perhaps the readme should show an example with with all the commandline options for dumdums like me :P

Side note: `-font "My Font:size=10.5"` does not work for me. What worked was `-font "My Font-10.5"`

> @pfr-dev
> 
> You can set the bar at the bottom by passing in `-bottom` on invocation like @realcharmer suggested, or you can write `all set-bottom` / `all set-top` into sandbar's stdin to change it at runtime. If you have a fifo setup like the one shown in the readme, you can do this:
> 
> ```shell
> echo 'all set-bottom' > "$MY_FIFO_PATH"
> ```

I'm just a little confused by the `all` and `status` 
I have `echo "all status $battery $volume $datetime" >"FIFO"` and thought that I needed to keep `status` in order to show the status's... therfore my first thought was to just add in `set-bottom` like so:
`echo "all set-bottom status $battery $volume $datetime" >"FIFO"` which didn't quite work. 

I've just set the command line option in the bar script which works fine.


> you can replace `all` with your specific output name if you want

Not quite sure what else I'd replace `all` with exactly. 

> As for your third question, the `-no-title` option exists to disable the window title, but nothing for the wm mode and layout. For now, you can probably just comment out these lines: https://github.com/kolunmi/sandbar/blob/main/sandbar.c#L412-L423 and you'll get the effect you want

That worked! would love for it to become a command line option in the future so I don't need to hack the source. 


One more question for now..

How can I set the `status-fg-color` and `status-bg-color` for my status output? I'd also like to have the option to set `urgent` and `critical` colors also for high temps, low battery etc.

Thanks!

**Comment by zistro** [comment]
> Side note: `-font "My Font:size=10.5"` does not work for me. What worked was `-font "My Font-10.5"`

This should be put into readme. Took me quite a long to figure it out.
